### PR TITLE
Implemented Ban Command

### PR DESCRIPTION
**Pull request by phillip-haydon**
Nuked my branch by mistake, had to re-implement :)

### CODE REVIEW

File: JabbR/Services/ChatService.cs [hunk 1]
```diff
@@ -1,953 +1,965 @@
...
+                                                                                {"lc", "Saint Lucia"},
+                                                                                {"li", "Liechtenstein"},
+                                                                                {"lk", "Sri Lanka"},
+                                                                                {"lr", "Liberia"},
+                                                                                {"ls", "Lesotho"},
+                                                                                {"lt", "Lithuania"},
+                                                                                {"lu", "Luxembourg"},
+                                                                                {"lv", "Latvia"},
+                                                                                {"ly", "Libya"},
+
```
Comment (phillip-haydon) [inline JabbR/Services/ChatService.cs:159]: I have no idea why this thinks everything changed... My VS is setup for spaces, github for windows shows only my addition. Github website shows everything changed....

**Comment by samandmoore** [comment]
@phillip-haydon what's happening with this? can you try again or should we just re-make these changes on our own?

**Comment by phillip-haydon** [comment]
I'm waiting for Windows 8 to come out so I can put that on my laptop and pull it on there... All 3 Windows 7 laptops, that single file does the same thing...

**Comment by phillip-haydon** [comment]
Closing because Chat Service is broken.

**Pull request closed without merging**